include/userobjects/MFEMGeneralUserObject.h

```diff
@@ -20,6 +21,11 @@ class MFEMGeneralUserObject : public GeneralUserObject
   // FIXME: Should this be marked as const if it is returning a non-const reference?
   MFEMProblem & getMFEMProblem() const { return _mfem_problem; }
 
+  /// Returns references to coefficients stored in the MFEMProblem PropertiesManager.
+  mfem::Coefficient & getScalarProperty(const std::string & name);
+  mfem::VectorCoefficient & getVectorProperty(const std::string & name);
+  mfem::MatrixCoefficient & getMatrixProperty(const std::string & name);
```

Comment:
I don't think we should call these `get*Property`, as that suggests they are related to materials. Call them something like `getFunctor` or `getCoefficient`.

Should we even bother having these? It does introduce a second way to access/achieve the same thing (even if it's only for convenience).